scene/3d/vehicle_body_3d.h

```diff
@@ -127,7 +127,8 @@ class VehicleWheel3D : public Node3D {
 	void set_use_as_steering(bool p_enabled);
 	bool is_used_as_steering() const;
 
-	bool is_in_contact() const;
+	bool is_in_contact() const;	
+	Node3D* get_contact_body();
```

Comment:
Coding style needs to be fixed, and this method could be const:
```suggestion
	Node3D *get_contact_body() const;
```